boards/common/msb-430/Makefile.include

```diff
@@ -1,8 +1,6 @@
 # set default port depending on operating system
 PORT_LINUX ?= /dev/ttyUSB0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
-# setup serial terminal
-include $(RIOTMAKE)/tools/serial.inc.mk
 
 # setup flash tool
 PROGRAMMER ?= olimex
```

Comment:
Just below `PROG_DEV` is used before being set, its a deferred evaluation so it would probably still work but could lead to weird cases.

Reply:
Fixed by #15533